src/components/Categories/style.scss

```diff
@@ -0,0 +1,138 @@
+.movie-list__categories {
+  max-width: 100%;
+  padding: 30px 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #e4e4e4;
```

Comment:
not necessarily but if you have extra time it will be better to move all color to separate scss file
and configure webpack to make them shared